DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/compute_units/sample.json

```diff
@@ -5,9 +5,9 @@
   "description": "Intel® FPGA tutorial showcasing a design pattern to enable the creation of compute units",
   "toolchain": ["dpcpp"],
   "os": ["linux", "windows"],
-  "targetDevice": ["FPGA"],
   "builder": ["ide", "cmake"],
   "languages": [{"cpp":{}}],
+  "targetDevice": [ "FPGA" ],
```

Comment:
I don't understand why this line needs to be moved?

Reply:
fixed